docs/_layouts/docs.html

```diff
@@ -3,193 +3,227 @@
-        <a href="{{ site.baseurl }}/docs/recipes.html">
-          Recipes
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/generated/api.html">
-          REST API
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/rest-api.html">
-          REST API (deprecated)
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/setup.html">
-          Setup Recommendations for Marathon
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/task-environment-vars.html">
-          Task Environment Variables
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/troubleshooting.html">
-          Troubleshooting
-        </a>
-      </li>
-    </ul>
-    <hr>
-    <h5>Development</h5>
-    <ul class="list-unstyled">
-      <li>
-        <a href="{{ site.baseurl }}/docs/deployment-design-doc.html">
-          Deployment Design
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/core-architecture.html">
-          New Core Architecture
-        </a>
-      </li>
-    </ul>
-    <hr>
-    <h5>Upgrading</h5>
-    <ul class="list-unstyled">
-      <li>
-        <a href="{{ site.baseurl }}/docs/upgrade/index.html">
-          Upgrading to a Newer Version
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/upgrade/06xto070.html">
-          0.6.x to 0.7.0
-        </a>
-      </li>
-    </ul>
-  </div>
-  <div class="col-sm-9">
-    {{ content }}
-  </div>
-</div>
+    <div class="col-sm-3">
+
+        <!-- <h5>Overview: What is Marathon and why is useful?</h5> -->
+
+        <h5>Getting Started</h5>
+        <ul class="list-unstyled">
+            <li>
+                <a href="{{ site.baseurl }}/docs/">
+                    Install Marathon
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/setup.html">
+                    Setup Recommendations
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/high-availability.html">
+                    High Availability Mode
+                </a>
+            </li>
+        </ul>
+
+        <h5>Application Deployment</h5>
+        <ul class="list-unstyled">
+            <li>
+                <a href="{{ site.baseurl }}/docs/application-basics.html">
+                    Application Basics
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/deployments.html">
+                    Application Deployments
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/command-line-flags.html">
+                    Command Line Flags
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/developing-vm.html">
+                    Run a Local Version of Marathon
```

Comment:
I suggest to put this under _Development_ – it not about using Marathon and not about administration either, mostly helpful for someone that wants to run in locally for development or testing purposes.